### PR TITLE
Modify event

### LINKED ISSUE #7: Edit Event
As a user, I should be able to edit an event.

### LINKED ISSUE #59: Run tests on the api folder

### LINKED ISSUE #60: Test registration api

### LINKED ISSUE #61: Test login api

### PR DESCRIPTION
**Pull request by zusoomro**

### CODE REVIEW

File: mobile/src/events/ModifyEvent.tsx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  Button,
+  SafeAreaView,
+  StyleSheet,
+} from "react-native";
+import DatePicker from "./DatePicker";
+import * as SecureStore from "expo-secure-store";
+
+const apiUrl = "http://localhost:8000";
+
+const CreateEvent: React.FC<{}> = ({ navigation, route }) => {
```
Comment (allysmith12) [inline mobile/src/events/ModifyEvent.tsx]: Why is this component called CreateEvent / is it supposed to be replacing/the same as the previous CreateEvent?

File: mobile/src/events/ScheduleNavigator.tsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { createStackNavigator } from "@react-navigation/stack";
 
 import CreateEvent from "./CreateEvent";
 import ScheduleHomePage from "./Schedule";
+import ModifyEvent from "./ModifyEvent";
```
Comment (allysmith12) [inline mobile/src/events/ScheduleNavigator.tsx:8]: I cannot seem to locate this component in your code?

**Review by allysmith12 — APPROVED** [review]
In the future it would helpful to have PR comments about what you did to accomplish the task in order to understand the code. Also, if we can try and separate out PR by functionality that would be super helpful because it's currently difficult to understand what is actually going on to accomplish Modify Event.

File: mobile/src/events/ModifyEvent.tsx (hunk 1 above)
Comment (zusoomro) [reply mobile/src/events/ModifyEvent.tsx]: Nope! Needs to be renamed.

**Pull request merged**